src/module/apps/sidebar/encounter-tracker.ts

```diff
@@ -180,19 +187,24 @@ export class EncounterTrackerPF2e<TEncounter extends EncounterPF2e | null> exten
             }
         }
 
+        if (trackerParent) {
+            trackerParent.insertBefore(tracker, trackerPlaceholder);
+            trackerPlaceholder.remove();
```

Comment:
Last thing, I think: try a `replaceWith` here.

Reply:
Didn't know that was a thing. It's been too long for me working with plain DOM APIs I guess.
Done, and thank you for your help here :)